frontend/src/Constants/SystemConstants.js

```diff
@@ -1,5 +1,6 @@
-export const BASE_URL = "";
-export const TOKEN_KEY = "";
-export const ROLE_KEY = "";
-export const NORMAL_USER = "";
-export const SUPER_USER = "";
+export const BASE_URL = "https://group4b6msapi.azurewebsites.net";
```

Comment:
Should be API_BASE_URL. We have two link, one for FE and one for BE. Then consider to name